documentation/2.5/themes/hugo-theme-learn/layouts/partials/menu.html

```diff
@@ -66,8 +66,9 @@ <h3>{{ if not $disableShortcutsTitle}}{{ T "Shortcuts-Title"}}{{ end }}</h3>
               {{ end }}
           {{ end }}
         </select>
```

Comment:
You still have some files that are reformatted that should not have changed.

Reply:
Yes, I don't know why IntelliJ touched this file, I only selected `Code -> Optimize Imports` option for operator module. I think these changes should be harmless.

Reply:
Revereted the doc changes in [27d2526](https://github.com/oracle/weblogic-kubernetes-operator/pull/2857/commits/27d2526417f5298d3de67b0d2c7eb825d636b01b).